examples/game_of_life/README.md

```diff
@@ -13,7 +13,7 @@ in a graph.
 
 ### Required Software
 
-- InstrumentStudio 2024 Q3 or later
+- InstrumentStudio 2024 Q4 or later
```

Comment:
We haven't been updating the per-example readmes every release, but I guess it makes sense to keep them consistent with this text in examples/README.md:

> The `examples` directory contains example measurements for use with InstrumentStudio 2025 Q1 or later.

Reply:
Updated.